src/C1T2X_OBU.py

```diff
@@ -39,17 +50,19 @@
 c1t2x_logger_formatter = logging.Formatter("[%(asctime)s.%(msecs)03d] %(levelname)s - %(message)s", datefmt= "%d-%b-%y %H:%M:%S")
 c1t2x_logger_handler.setFormatter(c1t2x_logger_formatter)
 c1t2x_logger.addHandler(c1t2x_logger_handler)
-# start logging
+# Start logging
 c1t2x_logger.info("\n---------------------------\nStarting C1T2X OBU Logger\n---------------------------")
 
-
-# initialize errors
+# Initialize error
 error = False
 
-# determine printing
-printData = True
+# Default to printing output
+if(args.print):
+	printData = True
+else:
+	printData = False
```

Comment:
`args.print` stores as `True`/`False`. Why can we not use `args.print` directly?